src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -126,5 +132,60 @@ public async Task<JObject> Resolve(string var_name, CancellationToken token)
             return rootObject;
         }
 
+        public async Task<JObject> Resolve(InvocationExpressionSyntax method, Dictionary<string, JObject> memberAccessValues, CancellationToken token)
+        {
+            JObject rootObject = null;
+
+            var expr = method.Expression;
+            var methodName = "";
+            if (expr is MemberAccessExpressionSyntax)
+            {
+                MemberAccessExpressionSyntax memberAccessExpressionSyntax = expr as MemberAccessExpressionSyntax;
+                rootObject = await Resolve(memberAccessExpressionSyntax.Expression.ToString(), token);
+                methodName = memberAccessExpressionSyntax.Name.ToString();
+            }
+            if (rootObject != null)
+            {
+                DotnetObjectId.TryParse(rootObject?["objectId"]?.Value<string>(), out DotnetObjectId objectId);
+                var typeId = await proxy.sdbHelper.GetTypeIdFromObject(sessionId, int.Parse(objectId.Value), true, token);
```

Comment:
proxy.sdbHelper could be local variable sdbHelper here for readability. Also I wonder if that's private field since it has lowercase name.